src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -115,77 +109,76 @@
-
-    } else {
-      System.out.println("Arguments\n"
-          + "   " + TRIPLEA_GAME_PROPERTY + "=<FILE_NAME>\n"
-          + "   " + TRIPLEA_SERVER_PROPERTY + "=true\n"
-          + "   " + TRIPLEA_CLIENT_PROPERTY + "=true\n"
-          + "   " + TRIPLEA_HOST_PROPERTY + "=<HOST_IP>\n"
-          + "   " + TRIPLEA_PORT_PROPERTY + "=<PORT>\n"
-          + "   " + TRIPLEA_NAME_PROPERTY + "=<PLAYER_NAME>\n"
-          + "   " + LobbyServer.TRIPLEA_LOBBY_PORT_PROPERTY + "=<LOBBY_PORT>\n"
-          + "   " + LOBBY_HOST + "=<LOBBY_HOST>\n"
-          + "   " + LOBBY_GAME_COMMENTS + "=<LOBBY_GAME_COMMENTS>\n"
-          + "   " + LOBBY_GAME_HOSTED_BY + "=<LOBBY_GAME_HOSTED_BY>\n"
-          + "   " + HttpProxy.PROXY_HOST + "=<Proxy_Host>\n"
-          + "   " + HttpProxy.PROXY_PORT + "=<Proxy_Port>\n"
-          + "   " + Memory.TRIPLEA_MEMORY_SET + "=true/false <did you set the xmx manually?>\n"
-          + "\n"
-          + "if there is only one argument, and it does not start with triplea.game, the argument will be \n"
-          + "taken as the name of the file to load.\n" + "\n" + "Example\n"
-          + "   to start a game using the given file:\n"
-          + "\n" + "   triplea /home/sgb/games/test.xml\n" + "\n" + "   or\n" + "\n"
-          + "   triplea triplea.game=/home/sgb/games/test.xml\n" + "\n" + "   to connect to a remote host:\n" + "\n"
-          + "   triplea triplea.client=true triplea.host=127.0.0.0 triplea.port=3300 triplea.name=Paul\n" + "\n"
-          + "   to start a server with the given game\n" + "\n"
-          + "   triplea triplea.game=/home/sgb/games/test.xml triplea.server=true triplea.port=3300 triplea.name=Allan"
-          + "\n"
-          + "   to start a server, you can optionally password protect the game using triplea.server.password=foo");
-    }
-  }
 
+  /**
+   * Launches the "main" TripleA gui enabled game client.
+   * No args will launch a client, additional args can be supplied to specify additional behavior.
+   * Warning: game engine code invokes this method to spawn new game clients.
+   */
   public static void main(final String[] args) {
     ErrorConsole.getConsole();
+    if (!ArgParser.handleCommandLineArgs(args, COMMAND_LINE_ARGS)) {
+      usage();
+      return;
+    }
     // do after we handle command line args
     Memory.checkForMemoryXMX();
 
-    SwingUtilities.invokeLater(() -> LookAndFeel.setupLookAndFeel());
+    SwingUtilities.invokeLater(LookAndFeel::setupLookAndFeel);
     showMainFrame();
-    new Thread(() -> setupLogging(GameMode.SWING_CLIENT)).start();
+    new Thread(GameRunner::setupLogging).start();
     HttpProxy.setupProxies();
     new Thread(GameRunner::checkLocalSystem).start();
-    new Thread(() -> checkForUpdates()).start();
-    handleCommandLineArgs(args, COMMAND_LINE_ARGS, GameMode.SWING_CLIENT);
+    new Thread(GameRunner::checkForUpdates).start();
+
+    final String version = System.getProperty(TRIPLEA_ENGINE_VERSION_BIN);
```

Comment:
Not a big deal, but this block of code seems like a good candidate for a method extraction.